src/docs/sdk/sessions.mdx

```diff
@@ -308,6 +308,16 @@ with one difference: sessions transitioned to `exited` have their session
 duration averaged. This lets Sentry show you the duration of non crashed
 sessions over time.
 
+## Alerts
+<Alert title="WIP" level="info">
+  Session based alerting is currently in early access.
+</Alert>
+
+Trigger an alert when a specified threshold of sessions experience an issue. 
+Create a new [issue alert](https://docs.sentry.io/product/alerts/alert-types/#issue-alerts) and select the "When" condition 
```

Comment:
Feels like this can use a screenshot?

Reply:
added a screenshot

Reply:
I think this is fine, but I kinda feel like the wording can be improved here. 

Maybe something like:
"Trigger an alert when an issue affects a specified percent of sessions. "